pkg/accounting/tally/tally_test.go

```diff
@@ -159,123 +162,144 @@ func TestCalculateNodeAtRestData(t *testing.T) {
 }
 
 func TestCalculateBucketAtRestData(t *testing.T) {
-	t.Skip("TODO: this test is flaky")
+	var testCases = []struct {
+		name         string
+		projectID    string
+		segmentIndex string
+		bucketName   string
+		objectName   string
+		inline       bool
+		last         bool
+	}{
+		{"inline, same project, same bucket", "mockProjectID", "l", "mockBucketName", "mockObjectName", true, true},
+		{"remote, same project, same bucket", "mockProjectID", "s0", "mockBucketName", "mockObjectName1", false, false},
+		{"last segment, same project, different bucket", "mockProjectID", "l", "mockBucketName1", "mockObjectName2", false, true},
```

Comment:
why do we want to specifically test the last segment here? just wondering.

Reply:
because the tally code has different behavior if its the last segment. re: https://github.com/storj/storj/blob/master/pkg/accounting/bucketstats.go#L70